nifi-nar-bundles/nifi-standard-services/nifi-hbase-client-service-api/src/main/java/org/apache/nifi/hbase/HBaseClientService.java

```diff
@@ -40,6 +40,18 @@
             .addValidator(new ConfigFilesValidator())
             .build();
 
+    PropertyDescriptor KERBEROS_PRINCIPAL = new PropertyDescriptor.Builder()
+            .name("Kerberos Principal")
+            .description("Principal of user writing to hbase")
+            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
+            .build();
+
+    PropertyDescriptor KERBEROS_KEYTAB = new PropertyDescriptor.Builder()
+            .name("Kerberos Keytab")
```

Comment:
For new `PropertyDescriptor`s, please use a unique permanent identifier string of the form `kerberos-keytab` for the `name` property and use the `displayName` property to hold UI values (can change in the future, internationalization, etc.). The `name` property is used for object resolution, and changing the value in the future to respond to user feedback will break backward compatibility with existing flows.